### ISSUE TITLE
[simclasses] better printing for I3MMCTrack (Trac #1874)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1866]
[simclasses] better printing for I3MMCTrack. because the memory address really means nothing to me. owner: olivas

Migrated from https://code.icecube.wisc.edu/ticket/1874
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:24", 
    "description": "[simclasses] better printing for I3MMCTrack. because the memory address really means nothing to me. owner: olivas", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067204154158", 
    "component": "combo reconstruction", 
    "summary": "[simclasses] better printing for I3MMCTrack", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-10-01T14:53:02", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "rumor / allegation"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20161004T13:05:28: 
* olivas changed resolution from "" to "fixed"
* olivas commented:

> In changeset 150498:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="150498"
> better printing for I3MMCTrack and I3MMCTrackList. closes #1866.
> ```


* olivas changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:24: 
* olivas changed milestone from "October 2016" to ""
* olivas commented:

> Milestone October 2016 deleted
